main.tf

```diff
@@ -90,6 +90,11 @@ resource "cloudamqp_alarm" "disk_alarm" {
   recipients      = local.recipients
 }
 
+resource "cloudamqp_rabbitmq_configuration" "rabbitmq_config" {
```

Comment:
Maybe a better solution would be to not have a default on the variable and add a count to this resource and only add it if the variable is set?

Reply:
Problem is that this can't be set per consumer/queue, acc. to documentation no other way obviously, don't like that either.
30 mins is Rabbit's default value.
But anyway, I'll change it